.github/workflows/pytest.yml

```diff
@@ -54,11 +54,8 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
 
-      # sphinxcontrib-applehelp added due to https://github.com/coddingtonbear/python-measurement/issues/75.
-      # To remove once the issue is resolved.
       - name: Install dependencies
         run: |
-          python -m pip install sphinxcontrib-applehelp==1.0.2
```

Comment:
Is it fixed in this branch?

Reply:
You are right, I wrongly noticed the version of `measurement` lib.

Decided to update already this library, but if shoudln't then I will revert.